### DISCUSSION TITLE
Multiple Crud Tables and/or Forms in one page

### DISCUSSION THREAD

**Discussion by leorae** [TiagoSilvaPereira/vemto-issues#225] [Ideas & Suggestions]
It would be nice if Vemto will allow multiple Crud in one page with nested tables.

**Comment by TiagoSilvaPereira** [comment]
Hi @leorae I'm not sure I understand but seems like you are saying about Master Detail... Vemto already supports it for hasMany relastionships

**Reply by leorae** [reply]
You can close this now. 😅

**Reply by TiagoSilvaPereira** [reply]
Awesome!
